source/features/rgh-improve-new-issue-form.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import clearCacheHandler from '../helpers/clear-cache-handler';
+import {isRefinedGitHubRepo} from '../github-helpers';
+
+function init(): false | void {
+	if (new URL(location.href).searchParams.get('template') !== '1_bug_report.yml') {
+		return false;
+	}
+
+	const {version} = browser.runtime.getManifest();
+	select('input#issue_form_extension_version')!.value = version;
+	select('input[id*="extension_cache"]')!.parentElement!.after(
+		<button
+			className="btn"
+			type="button"
+			onClick={clearCacheHandler as unknown as React.MouseEventHandler}
+		>
+			Clear cache
+		</button>,
+	);
```

Comment:
Let's also reword [the template](https://github.com/refined-github/refined-github/blame/6c6e2799cd1499f6e629cf9764e0e23b3fd19ce5/.github/ISSUE_TEMPLATE/1_bug_report.yml#L17) a bit:

```diff
- The "Clear cache" button can be found at the bottom of the options page.
+ The "Clear cache" button below can also be found at the bottom of the options page.
```

Reply:
How about dropping this line altogether?

Reply:
That could work, but will the user think the button is only available on the bug report form? 😬

Reply:
By the way we should bring that debugging box higher and drop the "warning" styles probably, so more people will be aware of it.